### ISSUE TITLE
error on macos with Apple Silicon

### ISSUE THREAD

**Issue by haaanz** [sinedie/SRTranslator#5]
I got this error when I tried to run on mac with command `python3 -m srtranslator ...`
Tried to install geckodriver: `brew install geckodriver` without success though

```
ERROR:webdriverdownloader.webdriverdownloader:Error, unable to determine correct filename for 64bit macos
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/srtranslator/translators/selenium_utils.py", line 47, in create_driver
    driver = webdriver.Firefox(proxy=proxy)
  File "/opt/homebrew/lib/python3.10/site-packages/selenium/webdriver/firefox/webdriver.py", line 197, in __init__
    super().__init__(command_executor=executor, options=options, keep_alive=True)
  File "/opt/homebrew/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py", line 288, in __init__
    self.start_session(capabilities, browser_profile)
  File "/opt/homebrew/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py", line 381, in start_session
    response = self.execute(Command.NEW_SESSION, parameters)
  File "/opt/homebrew/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py", line 444, in execute
    self.error_handler.check_response(response)
  File "/opt/homebrew/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py", line 249, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.SessionNotCreatedException: Message: Expected browser binary location, but unable to find binary in default location, no 'moz:firefoxOptions.binary' capability provided, and no binary flag set on the command line


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/homebrew/Cellar/python@3.10/3.10.8/Frameworks/Python.framework/Versions/3.10/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/opt/homebrew/Cellar/python@3.10/3.10.8/Frameworks/Python.framework/Versions/3.10/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/opt/homebrew/lib/python3.10/site-packages/srtranslator/__main__.py", line 78, in <module>
    translator = DeeplTranslator()
  File "/opt/homebrew/lib/python3.10/site-packages/srtranslator/translators/deepl.py", line 55, in __init__
    self.driver = create_driver(proxy)
  File "/opt/homebrew/lib/python3.10/site-packages/srtranslator/translators/selenium_utils.py", line 51, in create_driver
    gdd.download_and_install()
  File "/opt/homebrew/lib/python3.10/site-packages/webdriverdownloader/webdriverdownloader.py", line 174, in download_and_install
    filename_with_path = self.download(version,
  File "/opt/homebrew/lib/python3.10/site-packages/webdriverdownloader/webdriverdownloader.py", line 129, in download
    download_url = self.get_download_url(version, os_name=os_name, bitness=bitness)
  File "/opt/homebrew/lib/python3.10/site-packages/webdriverdownloader/webdriverdownloader.py", line 324, in get_download_url
    raise RuntimeError(info_message)
RuntimeError: Error, unable to determine correct filename for 64bit macos
```

**Comment by sinedie** [comment]
Hi, didn't try it in mac. The geckodriver is not installed and the library i used for automate this have this issue open: https://github.com/leonidessaguisagjr/webdriverdownloader/issues/18, which is what you are facing.

I will try another webdriver downloaders, but right now, you can use translatepy or use your own webdriver (maybe with tor service) or install geckodriver manually and add it to the system path

**Comment by sinedie** [comment]
Check the issue pointed in the last comment, but like a workaround you could run the next script -just once, no need to be every time you want to translate- to download and install geckodriver.

```
    from webdriverdownloader import GeckoDriverDownloader

    gdd = GeckoDriverDownloader()
    gdd.download_and_install(os_name='macos', bitness='-aarch64')
```

I did not try this as I don't own a mac.... so hope it works

**Comment by tinof** [comment]
I have the same problem with Macbook Pro M1.

So far I've tried to run the code @sinedie  described above (runs ok).
Also installed `brew install geckodriver` successfully.

Still the same error message remains (RuntimeError) when `python3 -m srtranslator ...` is executed

Interestingly enough, on the folder I run this I get new geckodriver.log file with following output:
`1673792652262   geckodriver     INFO    Listening on 127.0.0.1:50969`

I have tried to poke around the code leading to this error, but without success so far.
